package.json

```
@@ -113,19 +113,23 @@
     "@ckeditor/ckeditor5-slash-command": "40.0.0",
     "@ckeditor/ckeditor5-template": "40.0.0",
     "@ckeditor/ckeditor5-track-changes": "40.0.0",
+    "@ocular-d/vale-bin": "2.29.1",
```

It seems that only recent changes in this package are Vale version bumps, so it should be safe to follow them.

```suggestion
    "@ocular-d/vale-bin": "^2.29.1",
```